package proto_ff;

import "yd_fieldoptions.proto";


message E_DupBplevel
{
	optional int32 m_lv = 1[(yd_fieldoptions.field_cname) = "等级"];
	repeated int64 m_bgdup = 2[(yd_fieldoptions.field_cname) = "八卦本", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_DupBplevel
{
	repeated E_DupBplevel E_DupBplevel_List = 1[(yd_fieldoptions.field_arysize)=1100];
}

message E_DupDup
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "副本ID"];
	optional int32 m_scripid = 2[(yd_fieldoptions.field_cname) = "刷怪脚本ID"];
	optional int32 m_scriptype = 3[(yd_fieldoptions.field_cname) = "刷怪脚本类型"];
	optional int32 m_duplevel = 4[(yd_fieldoptions.field_cname) = "副本等级"];
	optional int32 m_groupid = 5[(yd_fieldoptions.field_cname) = "所属副本组ID"];
	optional int32 m_layer = 6[(yd_fieldoptions.field_cname) = "层级"];
	optional int32 m_duplicatetype = 7[(yd_fieldoptions.field_cname) = "副本内容类型"];
	optional int64 m_mapid = 8[(yd_fieldoptions.field_cname) = "场景ID"];
	optional int32 m_levellimit = 9[(yd_fieldoptions.field_cname) = "等级要求下限"];
	optional int32 m_pre = 10[(yd_fieldoptions.field_cname) = "前置副本要求"];
	optional int32 m_isautofight = 11[(yd_fieldoptions.field_cname) = "是否允许自动战斗"];
	optional string m_condition = 12[(yd_fieldoptions.field_cname) = "副本胜利条件类型", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_revivemaxtimes = 13[(yd_fieldoptions.field_cname) = "复活次数上限"];
	optional int32 m_teamobjectid = 14[(yd_fieldoptions.field_cname) = "组队目标ID"];
	optional string m_gradecondition = 15[(yd_fieldoptions.field_cname) = "星级时间", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 m_sweeprewardboxid = 16[(yd_fieldoptions.field_cname) = "扫荡奖励BOXID"];
	optional int32 m_sweeprewardexp = 17[(yd_fieldoptions.field_cname) = "扫荡经验奖励"];
	optional string m_trackpoint = 18[(yd_fieldoptions.field_cname) = "副本寻路点", (yd_fieldoptions.field_bufsize) = 60];
	optional string m_pathofmonster = 19[(yd_fieldoptions.field_cname) = "副本寻路点对应怪物", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_ispatrol = 20[(yd_fieldoptions.field_cname) = "寻路前是否巡敌"];
	optional int32 m_sweeptype = 21[(yd_fieldoptions.field_cname) = "扫荡类型"];
	repeated int32 m_box = 22[(yd_fieldoptions.field_cname) = "奖励9999组", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_DupDup
{
	repeated E_DupDup E_DupDup_List = 1[(yd_fieldoptions.field_arysize)=80];
}

message E_DupTowerreward
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_cond = 3[(yd_fieldoptions.field_cname) = "领取条件"];
	optional int32 m_boxid = 4[(yd_fieldoptions.field_cname) = "box"];
}


message Sheet_DupTowerreward
{
	repeated E_DupTowerreward E_DupTowerreward_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_DupGroup
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "副本组ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "副本组名", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_privilegetype = 3[(yd_fieldoptions.field_cname) = "特权类型"];
	optional int32 m_dynamiclevel = 4[(yd_fieldoptions.field_cname) = "是否动态等级"];
	optional int32 m_grouptimestype = 5[(yd_fieldoptions.field_cname) = "组次数类型"];
	optional int32 m_grouptimes = 6[(yd_fieldoptions.field_cname) = "次数上限值"];
	optional int32 m_enteritemid = 7[(yd_fieldoptions.field_cname) = "副本进入道具"];
	optional int32 m_sweepcondition = 8[(yd_fieldoptions.field_cname) = "扫荡条件"];
	optional int32 m_mergecondition = 9[(yd_fieldoptions.field_cname) = "合并条件"];
	optional int32 m_additemid = 10[(yd_fieldoptions.field_cname) = "增加次数物品ID"];
	optional int32 m_buytype = 11[(yd_fieldoptions.field_cname) = "购买次数消耗货币"];
	optional int32 m_buyitemid = 12[(yd_fieldoptions.field_cname) = "购买次数消耗物品"];
	optional string m_buycost = 13[(yd_fieldoptions.field_cname) = "购买进入次数消耗", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_countdown = 14[(yd_fieldoptions.field_cname) = "倒计时时间"];
	optional int32 m_enddown = 15[(yd_fieldoptions.field_cname) = "结束倒计时"];
	optional int32 m_accountdown = 16[(yd_fieldoptions.field_cname) = "结算倒计时"];
	optional int32 m_funcionid = 17[(yd_fieldoptions.field_cname) = "功能ID"];
}


message Sheet_DupGroup
{
	repeated E_DupGroup E_DupGroup_List = 1[(yd_fieldoptions.field_arysize)=20];
}
